service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,308 @@
+            <parameter name="statusId" default-value="orderPlaced"/>
+            <parameter name="productStoreId"/>
+            <parameter name="placedDate" required="true"/>
+            <parameter name="approvedDate"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+        </out-parameters>
+        <actions>
+            <!--Creating Orders in the OrderHeader -->
+            <service-call name="create#mantle.order.OrderHeader" in-map="context" out-map="context"/>
+        </actions>
+    </service>
+
+    <!--    Added Order Items-->
+
+    <service verb="add" noun="OrderItems" displayName="Add an order">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+            <parameter name="partName"/>
+            <parameter name="facilityId" required="true"/>
+            <parameter name="shipmentMethodEnumId" default-value="ShMthGround"/>
+            <parameter name="customerPartyId"/>
+            <parameter name="item_details" type="List">
+                <parameter name="orderMap" type="Map">
+                    <parameter name="productId" required="true"/>
+                    <parameter name="itemDescription" required="true"/>
+                    <parameter name="quantity" required="true"/>
+                    <parameter name="unitAmount" required="true"/>
+                </parameter>
+            </parameter>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+            <parameter name="orderPartSeqId"/>
+        </out-parameters>
+        <actions>
+            <!-- finding orderId from orderheader-->
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderList">
+                <econdition field-name="orderId"/>
+            </entity-find>
+            <if condition="orderList">
+                <service-call name="create#mantle.order.OrderPart" in-map="context" out-map="context"/>
+            </if>
+            <!-- finding orderId and customerId from orderPart-->
+            <entity-find entity-name="mantle.order.OrderPart" list="orderList">
+                <econdition field-name="orderId"/>
+                <econdition field-name="customerPartyId"/>
+            </entity-find>
+            <set field="productId" from="item_details.productId"/>
+            <set field="unitAmount" from="item_details.unitAmount"/>
+            <entity-find entity-name="mantle.product.Product" list="productList">
+                <econdition field-name="productId"/>
+            </entity-find>
+            <iterate list="productList" entry="product">
+                <set field="productId" from="product.productId"/>
+            </iterate>
+            <iterate list="orderList" entry="order">
+                <set field="orderId" from="order.orderId"/>
+            </iterate>
+            <service-call name="create#mantle.order.OrderItem" in-map="context" out-map="context"/>
```

Comment:
We must create orderItem for all the items in the item_details list. Can you check to iterate over the list and create OrderItem for each item?